uiowa/uiowa
# Duplicate views result with multiple arguments on people_list_block and article_list_block

Closes #4267 

Reduced duplicates in exposed filters - went with lightest fix

NOTE: this affects article list block - multiple people have multiple tags & people list block - when people have multiple research areas in addition to multiple tags


# How to test


1. checkout branch
2. blt ds --site=sandbox.uiowa.edu
3. go to https://sandbox.local.drupal.uiowa.edu/content-same-name and make sure no duplicate entries. Be sure to add new people with tags and add a pager (added: duplicate tag, duplicate bug test)
4.  Also test person research areas - looks like this was affected by same bug (added 'duplicate research area and duplicate bug test in prod - can filter on those)
NOTE: added research areas to prod - so can filter by them (duplicate research area and duplicate bug test)
4. Do same for articles list block too (added: bug test, same name)
